lib/sonice/public/script.js

```diff
@@ -116,6 +116,19 @@ $(function() {
     })
   }
 
+  // Toggle the playpause button on the remote button widget
+  var togglePlayPause = function(toggle_playpause_button){
```

Comment:
Really???  

But I couldn't find an existing code anywhere in the script that demonstrates good camel-cased letters.

Would you be expecting something like <b>'togglePlayPauseButton'<b>, perhaps?